src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -294,6 +314,23 @@ private static CommandLineProgram extractCommandLineProgram(final String[] args,
         return null;
     }
 
+    // Return a list of (usually Picard) classes which we want to mask from the list of runnable tools,
+    // usually because want to favor the like-named GATK tool. Any CommandLineProgram that shares a simple class
+    // name with a Picard tool must be on this list. Failure to mask it out will result in name collisions of
+    // simple class names in Main.
+    private static Set<String> getMaskedToolClasses() {
+        final Set<String> maskedPicardToolClasses = new HashSet<>();
+
+        // First, add the GATK Picard executor shim, since it's only a shim, not a runnable tool, and has no
+        // CommandLineProgramProperties of it's own.
+        maskedPicardToolClasses.add("org.broadinstitute.hellbender.cmdline.PicardCommandLineProgramExecutor");
+
+        // Mask out picard GatherVcfs in favor of GATK GatherVcfs
+        maskedPicardToolClasses.add("picard.vcf.GatherVcfs");
```

Comment:
I would eliminate the masking of Picard's `GatherVcfs` entirely, and just rename the GATK version. Suggested name for the GATK version: `GatherVcfsCloud`